js/script.js

```diff
@@ -0,0 +1,41 @@
+const taskInput = document.getElementById("input-task");
+const addButton = document.getElementById("add-task");
+const taskContainer = document.getElementById("task-container");
+
+const reg = /[&<>"'/`]/ig;
+
+let taskList = [];
+
+const addTaskHandler = () => {
+    const taskTitle = taskInput.value.replace(reg, "").trim();
```

Comment:
you can create a sanitize function in different `utility.js` file and export from that. as you may need it in further tasks.